docs/examples/snek5000-tgv/src/snek5000_tgv/tgv.usr.f

```diff
@@ -93,7 +101,7 @@ subroutine userchk()
 c      iostep_full = iostep
 c      call full_restart_save(iostep_full)
 
-      if (mod(istep,10).ne.0) return
+      if (mod(istep,4).ne.0) return
```

Comment:
This has to be improved! This should be every x equation time, with x potentially a user parameter.

Reply:
I'll work on it.

Reply:
In Fluidsim, we use the condition:

```python
if self.sim.time_stepping.t + 1e-15 // self.period_save > self.t_last_save // self.period_save
```

You could save for the first step (`istep == 0`) and then define `t_last_save`. For the other time steps, you could use a condition equivalent to what is done in Fluidsim.

Reply:
It could also be good to also save for the last step. (If it is easy to detect.)